CuratedContent/ResourcesforOSPO.md

```diff
@@ -0,0 +1,36 @@
+# Resources for Open Source Program Offices
+
+<!--deck text start-->
+Are you starting an Open Source Program Office (OSPO) or want to ensure the success of an existing OSPO or an open source project? If so, checkout these resources to help.
+<!--deck text end-->
+
+#### Contributed by [Patricia Grubel](http://github.com/pagrubel)
+
+#### Publication date: March 28, 2023
+
+Resource information | Details
+:--- | :---
+Organization type  | Open Source Program Offices
+Websites  |  Github OSPO  - [Github site to enable success on github for new OSPOS](https://github.com/github/github-ospo)<br> OSPO Alliance - [Community Driven Open Source Good Governance](https://ospo-alliance.org/)<br>TODO Group - [Talk Openly Develop Openly, an open community for sharing knowledge for successful OSPOs](https://todogroup.org/)<br> Open Source Guides - [Guides for Open Source Projects](https://opensource.guide/)
+Focus | Success for Open Source Program Offices (OSPOs)
+
+In some organizations the governance and guidance for open source projects fall under the guize of an Open Source Program Office.
+The community is beginning to stand up resources to enable sharing knowledge and guidance for the creation and success of an OSPO.
+The resources outlined in this article will help an organization to create an OSPO and to aid in the success of existing OSPOs.
+The resources are also useful for anyone who manages, maintains and or guides open source projects.
+There are even resources for open source developers, especially in the [Open Source Guides](https://opensource.guide/).
+
+Github is one of the main sites for hosting open source projects.
+The purpose of [Github OSPO](https://github.com/github/github-ospo), a github site itself, is to help OSPOs be successful on github.
```

Comment:
```suggestion
The purpose of [Github OSPO](https://github.com/github/github-ospo), a GitHub site itself, is to help OSPOs be successful on GitHub.
```

I wasn't sure why GitHub was capitalized sometimes and not others. I may be missing something about its usage here though.